neuraxle/steps/flow.py

```diff
@@ -0,0 +1,113 @@
+from neuraxle.base import MetaStepMixin, BaseStep, ExecutionContext, DataContainer, NonTransformableMixin, \
+    ExecutionMode, NonFittableMixin, ResumableStepMixin
+
+
+class ResumableMetaStepMixin(ResumableStepMixin):
+    def should_resume(self, data_container: DataContainer, context: ExecutionContext) -> bool:
+        if isinstance(self.wrapped, ResumableStepMixin):
+            wrapped_context = context.push(self.wrapped)
+            return self.wrapped.should_resume(data_container, wrapped_context)
+        return False
+
+
+class TransformOnlyWrapper(
+    NonTransformableMixin,
```

Comment:
It's weird to inherit from `NonTransformableMixin`. Why do we need that here already?

Reply:
I know. this is why I created the ForceHandleMixin. I had to copy it in 4 pull reqeuests to resolve your comments.

Reply:
we need that so that we don't need to implement fit, or transform methods. I've never been a fan of these mixins...